script/vsts/platforms/templates/bootstrap.yml

```diff
@@ -2,7 +2,7 @@ steps:
   - pwsh: |
       # OS specific env variables
       if ($env:AGENT_OS -eq "Windows_NT") {
-        $env:NPM_BIN_PATH="C:/hostedtoolcache/windows/node/12.4.0/x64/npm.cmd"
+        $env:NPM_BIN_PATH="C:/npm/prefix/npm.cmd"
```

Comment:
Why have you changed this?

Reply:
This is the `npm` install that changes when you do a simple `npm install -g npm` on Windows CI.

(If you run `npm-windows-upgrade`, it changes `C:/hostedtoolcache/windows/node/[node.version.here]/x64/npm.cmd`. But with this PR, we don't run the redundant `npm-windows-upgrade` command anymore. This PR deletes that step. To get an updated/pinned version of npm, we now have to use the `C:/npm/prefix/npm.cmd` one.)

Reply:
Please add this to the PR description.

Reply:
> Please add this to the PR description.

Done.